.gitignore

```diff
@@ -71,6 +71,17 @@ coverage.xml
 # Measurement results
 eval*.out
 test*.out
+eval/results/
+
+# Extra test files
+test/extras/
+test/spantest.py
+
+# Error analysis
+eval/examples_iEC/
+
+# Various resource files
+src/reynir_correct/resources/
```

Comment:
This folder should not be gitignored! The iceloc_prep.json file is assumed to be located there, and it is loaded by pattern.py.